ansible/roles/ocp4-workload-acm-app-policies-lab/files/pull-secret.yaml

```diff
@@ -0,0 +1,28 @@
+$ANSIBLE_VAULT;1.1;AES256
+62646264666563363963346139366166353734356263373332393966623437363538666437323435
+6236353865626134663532643033613935636565366137640a643937623932366164626561663137
```

Comment:
as discussed, can you use a secret variable instead ?

`ocp4_workload_acm_app_policies_lab_pull_secret_content`

That will contain the content of your file, and then use is in the `copy` module

Reply:
I moved this to a variable.